spyder/plugins/explorer/widgets.py

```diff
@@ -851,7 +877,181 @@ def create_func(fname):
+            file_content.setText('')
+            cb = QApplication.clipboard()
+            cb.clear(mode=cb.Clipboard)
+            cb.setMimeData(file_content, mode=cb.Clipboard)
+        except Exception as e:
+            QMessageBox.critical(self,
+                                 _('File/Folder Type Error'),
+                                 _("Cannot copy this type of file(s)"
+                                     "/folder(s):\n\n") + to_text_string(e))
+
+    @Slot()
+    def save_file_clipboard(self, fnames=None):
+        """Paste file from clipboard into file/project explorer directory."""
+        if fnames is None:
+            fnames = self.get_selected_filenames()
+        if not isinstance(fnames, (tuple, list)):
+            fnames = [fnames]
+        if len(fnames) >= 1:
+            try:
+                selected_item = osp.commonpath(fnames)
+            except AttributeError:
+                #  py2 does not have commonpath
+                if len(fnames) > 1:
+                    selected_item = osp.normpath(
+                            osp.dirname(osp.commonprefix(fnames)))
+                else:
+                    selected_item = fnames[0]
+            if osp.isfile(selected_item):
+                parrent_path = osp.dirname(selected_item)
+            else:
+                parrent_path = osp.normpath(selected_item)
+            cb_data = QApplication.clipboard().mimeData()
+            if cb_data.hasUrls():
+                urls = cb_data.urls()
+                for url in urls:
+                    source_name = url.toLocalFile()
+                    base_name = osp.basename(source_name)
+                    if osp.isfile(source_name):
+                        try:
+                            while base_name in os.listdir(parrent_path):
+                                file_no_ext, file_ext = osp.splitext(base_name)
+                                end_number = re.search(r'\d+$', file_no_ext)
+                                if end_number:
+                                    new_number = int(end_number.group()) + 1
+                                else:
+                                    new_number = 1
+                                left_string = re.sub(r'\d+$', '', file_no_ext)
+                                left_string += str(new_number)
+                                base_name = left_string + file_ext
+                                destination = osp.join(parrent_path, base_name)
+                            else:
+                                destination = osp.join(parrent_path, base_name)
+                            shutil.copy(source_name, destination)
+                        except Exception as e:
+                            QMessageBox.critical(self, _('Error Pasting File'),
+                                                 _("Unsupported Copy "
+                                                   "Operation:\n\n")
+                                                 + to_text_string(e))

```

Comment:
Let's change this to

```python
_("Unsupported copy "
  "operation. The error was:\n\n")
  + to_text_string(e)
```


taking into account not to going beyond 80 columns, of course.